migrator/src/main/java/uk/nhs/digital/ps/migrator/task/GenerateImportContentTask.java

```diff
@@ -100,7 +101,7 @@ public void execute() {
             createCcgImportables(catalogStructure, rootClinicalIndicatorsFolder)
         );
         importableItems.addAll(
-            createSocialCareImportables(catalogStructure, rootClinicalIndicatorsFolder)
+            SocialCareImportables.create(catalogStructure, rootClinicalIndicatorsFolder)
```

Comment:
Thanks for not making the `GenerateImportContentTask.java` class any longer - I shoul've done the same with CCG :-)